modules/gltf/src/glb/glb-builder.js

```diff
@@ -69,7 +68,7 @@ export default class GLBBuilder {
   // By default packs JSON by extracting binary data and replacing it with JSON pointers
   addApplicationData(key, data, packOptions = {}) {
     const packedJson = !packOptions.nopack && packBinaryJson(data, this, packOptions);
```

Comment:
Suggest resolving it here instead of referencing option several times

```
const jsonData = packOptions.nopack ? data : packBinaryJson(data, this, packOptions);
```

Wonder if we should bite the bullet now and change the options API to `pack` instead of `nopack` and set it to false by default. Seems like a clearer design.